lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.scss

```diff
@@ -46,4 +46,15 @@
     &-start-form-actions {
         text-align: right !important;
     }
+
+    &-loading {
+        margin: auto;
+    }
+
+    &-loading-container {
+        min-height: 300px;
+        display: flex;
+        flex-direction: row;
```

Comment:
The default is `row`, therefore this can be omitted

Reply:
fixed in new commit